### PR TITLE
Added checking arguments in GUI level

### PR DESCRIPTION
**Pull request by wojciech-zagrajczuk**

### CODE REVIEW

File: gi_gui/MainFrame.cpp [hunk 1]
```diff
@@ -335,10 +335,26 @@ void MainFrame::OnSave(wxCommandEvent& event)
...
-    local->GetValue(), limit->GetValue(), rescale->GetValue(), plane->GetValue(), refSlice->GetValue(), tarSlice->GetValue(),
-    refA->GetValue(), refB->GetValue(), tarA->GetValue(), tarB->GetValue());
+  Parameters* parsedParameters;
+  try
+  {
+    parsedParameters = parser->resolveParameters(dims->GetValue(), dd->GetValue(), dta->GetValue(), refval->GetValue(),
+     local->GetValue(), limit->GetValue(), rescale->GetValue(), plane->GetValue(), refSlice->GetValue(), tarSlice->GetValue(),
+     refA->GetValue(), refB->GetValue(), tarA->GetValue(), tarB->GetValue());
+  }
+  catch (const invalid_argument& ia)
```
Comment (kroczi) [inline gi_gui/MainFrame.cpp]: Is this exception defined anywhere? Does anyone throw it?

File: gi_gui/MainFrame.cpp [hunk 2]
```diff
@@ -277,6 +279,10 @@ void MainFrame::OnReference(wxCommandEvent& event)
 
   refPath = openFileDialog.GetPath();
   setSomeTextCtrl(referenceConfim, "Reference Path: " + refPath.ToStdString());
+
```
Comment (patrykks) [inline gi_gui/MainFrame.cpp]: artful

File: gi_gui/MainFrame.cpp [hunk 3]
```diff
@@ -277,6 +279,10 @@ void MainFrame::OnReference(wxCommandEvent& event)
...
+  if(firstPhotoSet)
+    calculate->Enable();
+  firstPhotoSet = true;
```
Comment (patrykks) [inline gi_gui/MainFrame.cpp]: artful

Comment (wojciech-zagrajczuk) [inline gi_gui/MainFrame.cpp]: thx

**Pull request merged**